core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -500,6 +501,20 @@ public SELF withBuild(boolean build) {
         return self();
     }
 
+    /**
+     * Adds options to the docker-compose command, e.g. docker-compose --compatibility. Options must be space separated
+     * Options are raw strings to allow future options to be accommodated transparently.
+     *
+     * @return this instance, for chaining
+     */
+    public SELF withOptions(String options) {
```

Comment:
Just adding to the Set here would be enough.
String for `runWithCompose` can be constructed using Stream and `Collectors.joining`.

Reply:
Building on @kiview's suggestion to use a set, this also would mean that `withOptions` could take a varargs String, and just add all to the set. This would be quite nice semantically.